src/main/resources/io/vertx/ext/shell/shell.html

```diff
@@ -0,0 +1,155 @@
+<!doctype html>
+<title>Vert.x Shell</title>
+
+<h1>Vert.x Shell</h1>
+
+<style>
```

Comment:
I would say that it make it more complex and I think that users that wants to customize it, will copy this file and the resources associated with it and customize it.